packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -0,0 +1,364 @@
+<template>
+    <div
+        ref="cookieBanner"
+        :class="[$style['c-cookieBanner'], $style['c-cookieBanner-overlay'], { [$style['c-cookieBanner--hidden']]: hideBanner }]"
+        data-cookie-consent-overlay
+        data-test-id="cookieBanner-component"
+        :aria-hidden="hideBanner">
+        <div
+            v-if="!legacyBanner"
+            :class="[$style['c-cookieBanner-card'], { [$style['c-cookieBanner-ios']]: checkIosSafari() }]">
+            <div
+                :class="$style['c-cookieBanner-content']"
+                role="dialog"
+                aria-labelledby="cookieConsentLabel"
+                aria-modal="true">
+                <h2
+                    id="cookieConsentLabel"
+                    :class="$style['c-cookieBanner-title']">
+                    <a
+                        ref="cookieBannerHeadingFocus"
+                        name="cookieConsentTitle"
+                        tabindex="0"
+                        data-consent-title>
+                        {{ $t('mainTitle') }}
+                    </a>
+                </h2>
+
+                <p :class="$style['c-cookieBanner-text']">
+                    {{ $t('textLine1') }}
+                </p>
+                <p :class="$style['c-cookieBanner-text']">
+                    {{ $t('textLine2') }}
+                </p>
+                <p
+                    :class="$style['c-cookieBanner-text']">
+                    <i18n
+                        path="textLine3">
+                        <template #cookiePolicy>
+                            <a
+                                :href="$t('cookiePolicyLinkUrl')"
+                                target="_blank">
+                                <span>{{ $t('cookiePolicyLinkText') }}</span>
+                            </a>
+                        </template>
+                    </i18n>
+                </p>
+            </div>
+
+            <div :class="$style['c-cookieBanner-CTA']">
+                <button-component
+                    tabindex="0"
+                    role="button"
+                    is-full-width
+                    @click.native="acceptActions">
+                    {{ $t('acceptButtonText') }}
+                </button-component>
+                <button-component
+                    tabindex="0"
+                    role="button"
+                    button-type="ghost"
+                    is-full-width
+                    @click.native="nonAcceptActions">
```

Comment:
```suggestion
                <button-component
                    button-type="ghost"
                    is-full-width
                    @click="nonAcceptActions">
```
As I commented above button should look like this

Reply:
See above